src/preferences/configobject.cpp

```diff
@@ -306,6 +337,41 @@ double ConfigObject<ConfigValue>::getValue(
     return ok ? result : default_value;
 }
 
+template<>
+template<>
+mixxx::RgbColor::optional_t ConfigObject<ConfigValue>::getValue(
+        const ConfigKey& key, const mixxx::RgbColor::optional_t& default_value) const {
+    const ConfigValue value = get(key);
+    if (value.isNull()) {
```

Comment:
We could add an optional default value to the conversion functions from QVariant to handle this use case.

Reply:
Done: https://github.com/mixxxdj/mixxx/pull/2518/commits/683e4c4fa659088bc6e7e79aef79fde51138ff2c